nplab/instrument/camera/Andor/default_config.json

```diff
@@ -0,0 +1 @@
+{"ReadMode": 4, "AcquisitionMode": 1, "TriggerMode": 0, "Exposure": 0.01, "Shutter": [1, 0, 1, 1], "SetTemperature": -90, "CoolerMode": 0, "FanMode": 0, "cooler": 0}
```

Comment:
can exposure be 1s? most common value in our labs for Si referencing.

Reply:
Sure, I'm not using the default_config anyway. In that case, do you know what your shutter speeds are?

Reply:
I don't know, so probably setting it to (1,0,0,0) is most sensible..

Reply:
I think that'll fail for most shutters. Which could be good, since it'll force people to actively set their own configurations), but could also be a pain in the ass

Reply:
Ok, nvm, it seems that's what master does now anyway, so yeah `Shutter=(1, 0, 0, 0)` it is